src/main/java/de/fraunhofer/iais/eis/ids/connector/main/Main.java

```diff
@@ -97,6 +97,9 @@
     @Value("${logging.outgoing}")
     private String loggingOut;
 
+    @Value("${jwks.trustedHosts}")
```

Comment:
Does this work if multiple values are provided? How are they separated - by comma?

Reply:
As a comma seperated list, this works just like in the broker.